src/goto-symex/builtin_functions.cpp

```diff
@@ -355,6 +355,51 @@ void goto_symext::symex_printf(const expr2tc &, const expr2tc &rhs)
     cur_state->guard.as_expr(), cur_state->source, fmt.as_string(), args);
 }
 
+void goto_symext::symex_scanf(
```

Comment:
This could be merged into fscanf function

Reply:
Thanks for the suggestions. I think I should merge them to a function like `symex_input()` .